js/webpack.config.js

```diff
@@ -80,6 +80,8 @@ module.exports = (env) => {
                 },
                 // this will apply to both plain `.css` files
                 // AND `<style>` blocks in `.vue` files
+                // NOTE: css-loader version 4.2.1 fail to load query builder style.
+                //      Revert to v. 3.6.0 until issue is found.
```

Comment:
should be easy to find the issues in "query builder style" - delete all styling and add it back in chunks until issue is present

Reply:
Would be good if it was that simple. 😄 The problem is that component style is not loaded at all within the bundle, thus not available at all. Downgrade to version 3 fixes the issue.

Reply:
we should aim to support the latest technologies, otherwise old technologies may be incompatible with another/user packages

so better to find a functional example for 4.x and mimic it

Reply:
agree but not now!

Reply:
ok with opened issue

Reply:
I always look up for using the latest package release in the bundle. No need to open an issue for this.